src/pages/Dashboard/Dashboard.tsx

```diff
@@ -1,18 +1,28 @@
-import { PageSection, PageSectionVariants } from "@patternfly/react-core";
+import { PageSection, PageSectionVariants, Alert, Button } from "@patternfly/react-core";
 import React from "react";
-import { RouteComponentProps } from "react-router-dom";
+import { RouteComponentProps, useHistory } from "react-router-dom";
+import ArrowRightIcon from '@patternfly/react-icons/dist/js/icons/arrow-right-icon';
 import PastAnalysisTable from "../../components/pastAnalysis/PastAnalysisTable";
 import Wrapper from "../../containers/Layout/PageWrapper";
 
 type AllProps = RouteComponentProps;
 
 const DashboardPage: React.FC<AllProps> = () => {
 
+  const history = useHistory();
+
   return (
     <Wrapper>
       <PageSection className="page-body">
-        {/* <PageSection className="section-area" variant={PageSectionVariants.light}>
-        </PageSection> */}
+        <PageSection className="section-area" variant={PageSectionVariants.light}>
+          <Alert isInline variant="warning" title="Patient lookup designated to Create Analysis page">
```

Comment:
I just used the default Alert component properties for this notification box, but this whole container is pretty wide.
![image](https://user-images.githubusercontent.com/53355975/118150960-1f5af400-b3e1-11eb-9a90-9d4edd3b5586.png)